README.md

```diff
@@ -1,27 +1,162 @@
+   If so, the SIG will assign dedicated reviewers to oversee the proposal, collaborate with others SIGs as needed, and provide 
+   feedback or veto when necessary. The author and reviewers work towards merged VEP.
+
+   Each SIG (Compute, Network, Storage) should sign-off on the VEP with an LGTM from a SIG representative (chair or approver), 
+   even if the proposal doesn't affect them. The sign-off works here as acknowledgment of the changes.
+
+4. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the
+   project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community
+   support and a commitment to implementation.
+
+    - On the EF, approved VEPs will be announced to kubevirt-dev mailing list in order to draw attention to them.
+    - Each week a release team will check tracked VEP in order to assure activity.
+
+5. **Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with
+   the Kubernetes KEP process. It will ensure that each enhancement includes all the relevant information, including the
+   design and the state.
+
+### Responsibilities
+
+#### VEP Owner
 
 The VEP owner is responsible to update it as its development progresses, until it is fully mature (or deprecated).
+In addition, they are encouraged:
+
+- talk about the design in the unconference sessions to bring everyone on the same page about the problem, 
+  use-cases/design etc...
+- join the relevant sig calls for further discussions, and seeking for reviews.
+- ask for reviews on community calls if this is a cross-sig VEP.
+
+#### SIGs
+
+The responsibility of the SIGs is to do their best to help ensure sure the VEP is implemented, not diverging and that the 
+implementation is not lacking behind the VEP, following this non-exhaustive list meant as checklist:
+
+1. After Code Freeze, SIGs need to go over a tracking issue and perform the following checklist:
+   1. All PRs are merged into release branch
+   2. Docs PR is merged (plan review ahead of release if only placeholder is opened)
+   3. Verify that the Enhancement was implemented and doesn't need any update or exception
+   4. Track any bugs
+   5. Make sure the VEP's issue is tracking required PRs and Issues
+2. Weekly check-in on progress of the Enhancement and its implementation
+3. Coordinate SIGs, reviewers and approvers in order to progress the Enhancement
+
+### Release check-ins
+Both the release team and approvers of the VEPs are responsible for weekly check-ins, the outcomes of which will be posted on 
+the VEP's tracking issue. The following are the goals of the
+check-ins:
+
+1. Re-targeting of VEP - In case of implementation not converging, new blockers being discovered, pushback of community
+   or withdrawal of an approver the VEP needs to be re-targeted. The VEP needs to be updated with a new target. The SIGs
```

Comment:
```suggestion
1. Re-targeting of VEP - In case of implementation not converging, new blockers being discovered, pushback of community.
   or withdrawal of an approver, the VEP may need to be re-targeted to a different release. In this case, the VEP needs to be updated with the new target and the SIGs
```
Maybe?